cockpit/src/subscriptions-client.js

```diff
@@ -399,12 +267,12 @@ client.init = function() {
     /* ideally we could get detailed subscription info via DBus, but we
      * can't rely on this being present on all systems we work on
      */
-    service.subscribe(
+    legacyService.subscribe(
         { path: "/EntitlementStatus",
           interface: "org.freedesktop.DBUS.Properties",
           member: "PropertiesChanged"
         },
-        function(path, iface, signal, args) {
+        () => {
```

Comment:
I assume this is shorthand for a no-arg lambda, where we're ignoring the args because the function doesn't use them anyway...?

Reply:
@Ceiu, yeah, more-or-less: `() => {` is essentially the same as `function() {`, except that it doesn't define its own `this`, and if the function has only a single statement, there is an implicit return. More reading if curious: https://hacks.mozilla.org/2015/06/es6-in-depth-arrow-functions/